src/scipp/experimental/plotting/wrappers.py

```diff
@@ -36,15 +49,66 @@ def plot(obj: Union[VariableLike, Dict[str, VariableLike]]) -> Figure:
         - dict of Variables
         - dict of DataArrays
         - dict of numpy ndarrays
+    aspect:
+        Aspect ratio for the axes.
+    cbar:
+        Show colorbar in 2d plots if `True`.
+    crop:
+        Set the axis limits. Limits should be given as a dict with one entry per
+        dimension to be cropped. Each entry should be a nested dict containing scalar
+        values for `'min'` and/or `'max'`. Example:
+        `da.plot(crop={'time': {'min': 2 * sc.Unit('s'), 'max': 40 * sc.Unit('s')}})`
+    errorbars:
+        Show errorbars in 1d plots if `True`.
+    grid:
+        Show grid if `True`.
+    mask_color:
+        Color of masks in 1d plots.
+    norm:
+        Set to 'log' for a logarithmic y-axis (1d plots) or logarithmic colorscale
```

Comment:
Mention other option ('linear'?)? But maybe sphinx does this automatically if you use `Literal` in the type hint (see other comment).

Reply:
done